fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaGenerator.java

```diff
@@ -1183,13 +1373,16 @@ protected void addCodeSystems(PhysicalDataModel model) {
      * the token_value represents its logical_id. This approach simplifies query writing when
      * following references.
      *
+     * If sharding is supported, this table is distributed by token_value which unfortunately
+     * means that it cannot be the target of any foreign key constraint (which needs to use
+     * the primary key COMMON_TOKEN_VALUE_ID).
```

Comment:
this says its distributed by token_value, but down below i see
```
setDistributionType(DistributionType.REFERENCE)
```
maybe a stale comment?

Reply:
thanks. fixed.